Makefile

```diff
@@ -40,8 +49,10 @@ build-yarn-watch: build-solidity-types build-cargo
 
 .PHONY: build-cargo
 build-cargo: ## build cargo packages and create boilerplate JS code
-	cargo build --release --target wasm32-unknown-unknown
-	$(MAKE) -j 1 $(WORKSPACES_WITH_RUST_MODULES)
+# First compile Rust crates and create bindings
+	$(MAKE) -j 1 $(CRATES)
```

Comment:
Why do you want to execute one by one?

Reply:
When running multiple instances of `cargo`, like `cargo build first_crate`, `cargo build second_crate`, `cargo` uses some file-lock based approach to build crates only once. The approach seem to work on my Debian-based system but it fails when running the build as part of a Github Action.